MetaMask/metamask-extension
# feat: updated SSK version in e2e and added test for creating multiple…

## **Description**

Adds an e2e test where the account Snap suggests the same account name more than once. 
The expected behavior is that the default account names should have an increasing numeric suffix, e.g.:

- SSK Account
- SSK Account 2
- SSK Account 3

Dependency: https://github.com/MetaMask/snaps-registry/pull/735

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26378?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/548

## **Manual testing steps**

1. Install the [Snap Simple Keyring](https://metamask.github.io/snap-simple-keyring/latest/) and connect to the dapp
2. Create an account -> name of 1st account should be "SSK Account"
3. Create more accounts -> 2nd account should be "SSK Account 2" and so forth
4. Try renaming a numbered SSK Account to a higher number, the next account created should be the lowest possible available "SSK Account X" with X being the lowest available number 

## **Screenshots/Recordings**

https://github.com/user-attachments/assets/b2404a8f-dbe9-468f-9ca3-aac1e028acae

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
